python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/monitor_twamp.schema.yml

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../_schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  monitor_twamp:
+    documentation_options:
+      table: monitor-twamp
```

Comment:
this table must be included in the input variables markdown file. I think there is a monitoring section where it could fit.

Reply:
Fixed now, thanks!